qemu/tests/cfg/qemu_img.cfg

```diff
@@ -86,3 +86,33 @@
         - commit:  install setup image_copy unattended_install.cdrom
             no vmdk raw qed
             subcommand = commit
+        - amend:
+            only Host_RHEL.m7
+            subcommand = amend
+            start_vm = no
+            images += " test"
+            image_name_test = "images/test"
+            create_image = yes
+            create_image_image1 = no
+            force_create_image = yes
+            force_create_image_image1 = no
```

Comment:
for 3th question: After specify params "create_image_stg = yes" here, avocado frame will create an image for test instead of creating it in case.